test/helpers/kubectl.go

```diff
@@ -3129,8 +3134,30 @@ func (kub *Kubectl) CiliumPolicyAction(namespace, filepath string, action Resour
 	return kub.waitNextPolicyRevisions(podRevisions, timeout)
 }
 
+// DeleteAllPolicies deletes all CNPs and NPs in the given 'namespace', as well as all
+// (cluster-scoped) CCNPs and waits up until 'timeout' seconds for the policies to be deleted from
+// all Cilium endpoints.
+//
+// Does NOT return an error as this is to be used from AfterEach blocks.
+func (kub *Kubectl) DeleteAllPolicies(namespace string, timeout time.Duration) {
```

Comment:
`DeleteAllPoliciesSync` would instantly tell me that this method waits for policy deletion.